Blockstack/Classes/Blockstack.swift

```diff
@@ -9,24 +9,9 @@ import Foundation
 
 open class Blockstack {
```

Comment:
I understand that `Blockstack` class was created for future needs of:

- Dependency injection of `BlockstackInstance`
- Configuration of `BlockstackInstance`

Very good idea. This change will keep the planned functionality and greatly simplify the code.